frontend/src/layout/navigation/navigationLogic.ts

```diff
@@ -114,4 +151,14 @@ export const navigationLogic = kea<navigationLogicType<UserType, SystemStatus>>(
             actions.loadLatestVersion()
         },
     }),
+    urlToAction: ({ actions }) => ({
```

Comment:
Q: What is the goal here? Not obvious from the code + '*' urlToAction has proven itself to be super fragile (3 bugfixes within a week for it)

Reply:
Removing this based on discussion from https://github.com/PostHog/posthog/pull/3174#discussion_r571995331